mmf/utils/general.py

```diff
@@ -316,3 +316,11 @@ def assert_iterator_finished(iter):
         pass
     else:
         assert False
+
+
+def get_device():
```

Comment:
Even though this is small, it would still be good to write tests for this to catch any pytorch related regressions in future.